lib/msal-core/samples/react-sample-app/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Should we add some msal / react specific things to the read me ?

Reply:
still planning to make this change?

Reply:
Yeah, working on it now.